.github/workflows/checks.yml

```diff
@@ -0,0 +1,49 @@
+name: README Tests
+
+on:
+  push:
+    branches: [stable, dev, legacy-v3]
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  format_check:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - run: npm ci
+      - run: rustup target add wasm32-unknown-unknown
+      - run: npm run fmt
```

Comment:
Rust steps feel unnecessary, there no WASM on this project right?

Reply:
Haha you're right I copy-pasted stuff too quickly I think

Reply:
done